recipes/stackeddag/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "stackeddag" %}
+{% set version = "0.3.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/stackeddag-{{ version }}.tar.gz
+  sha256: 89404050364710f25188812248735fed34dae90c001e35e74bff7fc15a9f970b
+
+build:
```

Comment:
This package looks like it should be `noarch: python`. See:
https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-builds

Reply:
I did update the PR. Does this mean [grayskull](https://github.com/conda-incubator/grayskull) failed to infer that the package should be ``noarch: python``?